content/en/docs/Tasks/Troubleshooting/subscription.md

```diff
@@ -28,3 +28,17 @@ If the `Status` block does not provide enough information, check the [Catalog op
 ### Why does a single failing subscription cause all subscriptions in a namespace to fail?
 
 Each Subscription in a namespace acts as a part of a set of operators for the namespace - think of a Subscription as an entry in a python `requirements.txt`. If OLM is unable to resolve part of the set, it knows that resolving the entire set will fail, so it will bail out of the installation of operators for that particular namespace. Subscriptions are separate objects but within a namespace they are all synced and resolved together.
+
+### Subscription fails due to missing APIs
+Newer versions of kubernetes have deprecated and removed certain APIs that OLM supports. For example, the `apiextensions.k8s.io/v1beta1` GroupVersion is no longer available on kubernetes 1.22+ clusters. Operators that rely on v1beta1 CRDs will fail to install on 1.22+ clusters, since the v1beta API has been removed. See 
```

Comment:
WDYT about changing the tense of this sentence, e.g. "Newer versions of Kubernetes may deprecate and remove certain APIs that OLM supports".